common/src/main/java/com/wynntils/features/inventory/PreventMixedRerollsFeature.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.inventory;
+
+import com.google.common.primitives.Ints;
+import com.wynntils.core.components.Models;
+import com.wynntils.core.consumers.features.Feature;
+import com.wynntils.core.persisted.config.Category;
+import com.wynntils.core.persisted.config.ConfigCategory;
+import com.wynntils.mc.event.ContainerClickEvent;
+import com.wynntils.mc.event.ContainerSetContentEvent;
+import com.wynntils.mc.event.ContainerSetSlotEvent;
+import com.wynntils.mc.event.ItemTooltipRenderEvent;
+import com.wynntils.mc.event.MenuEvent;
+import com.wynntils.mc.event.ScreenClosedEvent;
+import com.wynntils.mc.event.SlotRenderEvent;
+import com.wynntils.models.items.WynnItem;
+import com.wynntils.models.items.items.game.GameItem;
+import com.wynntils.models.items.items.gui.IngredientPouchItem;
+import com.wynntils.models.items.properties.UnidentifiedItemProperty;
+import com.wynntils.utils.colors.CommonColors;
+import com.wynntils.utils.mc.McUtils;
+import com.wynntils.utils.render.buffered.BufferedRenderUtils;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import net.minecraft.ChatFormatting;
+import net.minecraft.network.chat.Component;
+import net.minecraft.world.inventory.AbstractContainerMenu;
+import net.minecraft.world.inventory.Slot;
+import net.minecraft.world.item.ItemStack;
+import net.neoforged.bus.api.EventPriority;
+import net.neoforged.bus.api.SubscribeEvent;
+
+@ConfigCategory(Category.INVENTORY)
+public class PreventMixedRerollsFeature extends Feature {
+    private static final String ITEM_IDENTIFIER_NAME = "\udaff\udff8\ue018";
+    private static final int[] ITEM_IDENTIFIER_SLOTS = {11, 12, 13, 14, 15, 20, 21, 22, 23, 24};
```

Comment:
This looks like the kind of "knowledge" that should be owned by a Model. I'm not sure which one, though. We don't seem to have an IdentifierModel; perhaps it is time to create one. Or do you have any better suggestions, @kristofbolyai?

Reply:
It could live in the container specific class for the Identifier, although we don't have one yet. I forgot what the superclass is called, but it's somewhere around ContainerModel.

Reply:
Yes, that is indeed the correct place. 

Look at e.g. `com.wynntils.models.containers.containers.SeaskipperContainer`; you need to create a similar `IdentifierContainer` (or if you like being verbose, `ItemIdentifierContainer`).